website/src/docs/guides/backups.md

```diff
@@ -36,11 +47,17 @@ To ensure a smooth restoration process, remember to:
 1. Log into the Tracking services you previously used.
 1. Download any extensions you've used in your backup.
 
-## Transferring Downloads to a New Installation
+Tracking Services used and Extensions previously used are stored in the Backup.
+Before starting to import the selected backup, the app will remind the user of these.
+
+### Transferring Downloads to a New Installation
+
+You can transfer downloaded series chapters from one version of **Tachiyomi** to another
+by correctly specifying the Download Folder Location
 
-You can transfer downloaded series chapters from one version of **Tachiyomi** to another.
+## Suggestions for Backups
 
-## Enabling Automatic Backups
+### Enable Automatic Backups
```

Comment:
```suggestion
### Enabling Automatic Backups
```
Other section titles are present participles so this one should be too

Reply:
Changing Sync to Syncing as well